connectors/filtering/validation.py

```diff
@@ -196,6 +196,12 @@ def validate(cls, rules):
 
 
 class BasicRulesSetSemanticValidator(BasicRulesSetValidator):
+    """Class validating, that a set of rules does not contain semantic duplicates.
+
+    A semantic duplicate is defined as two basic rules having the same values for `field`, `rule` and `value`.
+    Therefore, two basic rules are also seen as semantic duplicates, if their `policy` values differ.
```

Comment:
What happens in this case? Is the rule set declared as invalid?

Reply:
Added an explanation.